java/src/client/server/MockProxy.java

```diff
@@ -24,7 +24,9 @@ public MockProxy(){}
      * @param credentialsObject The information that needs to be added to the body of the HTTP request.
      * @return true if user is logged in; false otherwise
      * @pre username and password are not null
-     * @post If the passed­in (username, password) pair is valid,  1. The server returns an HTTP 200 success response with “Success” in the body.  2. The HTTP response headers set the catan.user cookie to contain the identity of the  logged­in player.  The cookie uses ”Path=/”, and its value contains a url­encoded JSON object of  the following form: { “name”: STRING, “password”: STRING, “playerID”: INTEGER }.  For  example, { “name”: “Rick”, “password”: “secret”, “playerID”: 14 }.    If the passed­in (username, password) pair is not valid, or the operation fails for any other  reason,  1. The server returns an HTTP 400 error response, and the body contains an error  message.
+     * @post If the passed­in (username, password) pair is valid, 
+     * @post 2. The HTTP response headers set the catan.user cookie to contain the identity of the  logged­in player.  
```

Comment:
You don't need `@post ` on each line of the paragraph - that will actually create multiple post-conditions.
Have each new line start with spaces to match up with the previous.